conf/defaults.ini

```diff
@@ -1011,3 +1011,10 @@ default_timezone = browser
 [expressions]
 # Enable or disable the expressions functionality.
 enabled = true
+
+[geomap]
+# Set the default base layer in GeoMaps plugin
```

Comment:
Can you also update: `sample.ini` and `configuraiton.ini`?

![image](https://user-images.githubusercontent.com/705951/126391709-92ce89fe-a90e-42b8-92c1-79efda9c1bf1.png)